File: New folder/Day8_Inheritance-2/A1_Wage_Manager_Salesperson/Person.cpp
#include"Person.h"

Person::Person()////////////////////////constructors
{
    age=0;
}
// Person::Person(Cstring nm, int age):name(nm)
// {
//     this->age=age;
// }
Person :: Person(const char * nm, int age):name(nm)
{
    this->age=age;
    // Cstring(nm);
}
Person::Person(Person &obj):name(obj.name)////////////////////////////////////copy constructor
{
    // name = obj.name;
    this->age=age;
}
void Person::display()/////////////////////////facilitators
{
    name.display();
    cout<<"Age: "<<age<<"\n";
}

File: Core_java_new/Day4_Package/office/staff/Person.java
package office.staff;

public class Person
{	
	private String name;
	public Person()
	{
		
		name= new String("Empty");
	}
	public Person(String name)
	{
		this.name=name;
	}
	
	public void display()
	{
		System.out.println("Name: "+name);
	}
	
}